syncthing_gtk/editordialog.py

```diff
@@ -256,7 +256,7 @@ def display_value(self, key, w):
 		elif isinstance(w, Gtk.ComboBox):
 			val = self.get_value(strip_v(key))
 			m = w.get_model()
-			for i in xrange(0, len(m)):
+			for i in range(0, len(m)):
```

Comment:
As far as I could tell, none of the usages of ```xrange()``` were over a large enough size that it would matter much if ```range()``` were used instead.  It would use a *tiny* bit more memory in Python 2, but in Python 3, ```range()``` is just like ```xrange()```.